Packages/com.unity.inputsystem/InputSystem/Devices/Touchscreen.cs

```diff
@@ -374,6 +374,7 @@ internal unsafe struct TouchscreenState : IInputStateTypeInfo
         [InputControl(name = "pressure", useStateFrom = "primaryTouch/pressure")]
         [InputControl(name = "radius", useStateFrom = "primaryTouch/radius")]
         [InputControl(name = "press", useStateFrom = "primaryTouch/phase", layout = "TouchPress", synthetic = true, usages = new string[0])]
+        [InputControl(name = "displayIndex", useStateFrom = "primaryTouch/displayIndex", format = "BYTE")] // added format to override the Pointer's USHT value
```

Comment:
Looks like a good fix. Thanks for adding the comment to say why format is added to make sure its not removed in future refactorings.